server/rest/src/test/java/org/infinispan/rest/resources/ProtobufResourceTest.java

```diff
@@ -139,11 +139,9 @@ public void addAndGetListOrderedByName() throws Exception {
 
       String personProto = getResourceAsString("person.proto", getClass().getClassLoader());
 
-      join(CompletionStages.allOf(
-            schemaClient.post("users", personProto),
-            schemaClient.post("people", personProto),
-            schemaClient.post("dancers", personProto)
-      ));
+      join(schemaClient.post("users", personProto));
+      join(schemaClient.post("people", personProto));
+      join(schemaClient.post("dancers", personProto));
```

Comment:
Are we human?
Or are we dancers?

Reply:
is this a review comment ? 👯 👯‍♀️

Reply:
Absolutely. I think so. To follow the lyrics of the song, the schema name should have been "humans", but I'll let this slide today :))